### DISCUSSION TITLE
Hide the entire task bar on a specific monitor

### DISCUSSION THREAD

**Discussion by battlepanda** [valinet/ExplorerPatcher#3437] [Ideas]
I'm currently utilizing a three-monitor setup, with one monitor primarily dedicated to media like movies. This particular monitor, measuring 1920x480, is uniquely suited for displaying content without the clutter of additional interface elements due to its limited vertical space. 

I propose a feature that allows this third monitor to display just the Windows desktop background, free of the standard taskbar, start menu, and system tray icons that are present on the other two primary monitors. This would maximize the valuable vertical real estate of the screen and enhance its functionality as a dedicated display monitor.

![fvp8rOHHG4](https://github.com/valinet/ExplorerPatcher/assets/12422772/e7acbbc4-f8fe-40d9-a4d3-7146b231c721)

**Comment by pyrates999** [comment]
Normally in any application playing video, like vlc, setting it to full screen mode puts it on top of whatever is on a monitor.  Including the taskbar there.  So this is not needed.

**Comment by battlepanda** [comment]
Yeah i certainly agree when it comes to VLC or even a browser (youtube etc) with the full screen button.

The idea is more something like Twitch when a browser is opened [maximised] but still has access to the core browser functions etc at the top, so not truly full screened.

**Reply by pyrates999** [reply]
Makes sense.

**Comment by dev-sts** [comment]
I would also like to see this feature implemented.
I also have 3 monitors and would like to display the taskbar on 2 of them.

In my case, however, the reason is that the 3rd monitor has an OLED panel and there would be a fundamental risk of burn-in due to the static content of the taskbar.
